avm/res/event-grid/domain/main.json

```diff
@@ -5,8 +5,8 @@
   "metadata": {
     "_generator": {
       "name": "bicep",
-      "version": "0.25.53.49325",
-      "templateHash": "7406933903897002331"
+      "version": "0.26.54.24096",
```

Comment:
I suggest to undo changes for all templates where **only** the version changed.

Reply:
done